doc/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    = -a -E
+SPHINXOPTS    = -W
```

Comment:
dummy question, how does this change the build process?

Reply:
I'll change this back before merging this, but:

- `-a -E` makes sphinx rebuild the docs each time instead of using the existing build artifacts
- `-W` makes sphinx turn any build warnings into build _errors_, stopping the build.